guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) If the learning rate is not <s>mentioend </s><strong>mentioned </strong>then an optimal learning rate is automatically deduced for training the model</p><p>(2) &nbsp;save the best model during training, which <s>if</s> <strong>is</strong> set to True,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>